recipe/meta.yaml

```diff
@@ -161,6 +161,7 @@ requirements:
     - readline
     - {{ pin_compatible("rw", max_pin="x.x") }}
     - singular
+    - sirocco
```

Comment:
It should be a `host` dependency, so `sagelib` is built against it. Not a `run dependency`.

Reply:
OK, I'll change it